VERSION

```diff
@@ -1 +1 @@
-v0.6.0
+cstor-integration-v0.6
```

Comment:
Lets not use the VERSION file for the branch image tag. Refer my comment above.

Reply:
ok, will change